charts/argo-cd/templates/_helpers.tpl

```diff
@@ -270,3 +274,10 @@ Create the name of the configmap to use
     {{ default "argocd-notifications-cm" .Values.notifications.cm.name }}
 {{- end -}}
 {{- end -}}
+
+{{/* 
+    Create Hostname helper -- Coreweave Use Only
+*/}}
+{{- define "coreweave.externalDnsName" -}}
+{{ printf "argocd.%s.ord1.ingress.coreweave.cloud" .Release.Namespace }}
```

Comment:
I think we should add an override value here. Something like ingress.overrideHostname.

Reply:
If we do that then we will have to deploy their own ingress controller as well (I believe), do we want to bundle it that way? Since this is the format for the kubemod rule.

Reply:
We can do w/out for now. If we get a request for is we can revisit